SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -101,6 +109,7 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject, public FRunnable
 	void StartDevelopmentAuth(FString DevAuthToken);
 	static void OnPlayerIdentityToken(void* UserData, const Worker_Alpha_PlayerIdentityTokenResponse* PIToken);
 	static void OnLoginTokens(void* UserData, const Worker_Alpha_LoginTokensResponse* LoginTokens);
+    void OnLoginTokens(const Worker_Alpha_LoginTokensResponse* LoginTokens);
```

Comment:
This name reads oddly as there is no verb. What has happened when this callback is called?

Reply:
Perhaps consider using `On...` for registering a callback and `Trigger...` for triggering callbacks.